docs/sdks/ios/methods.md

```diff
@@ -63,11 +88,36 @@ NSArray* subscriptionTopics = [CleverPush getSubscriptionTopics];
 Swift:
 
 ```swift
+// get available tags
 let channelTags = CleverPush.getAvailableTags()
 
-CleverPush.addSubscriptionTag("TAG_ID")
+// add/remove tag with action callback
+CleverPush.addSubscriptionTag("TAG_ID", callback: { tagId in
```

Comment:
Nice, can you also leave the "old" method without callback in the docs? So the user knows both ways are possible.

Reply:
I have already added without callback methods.